pipelines/commands/syncstatus_test.go

```diff
@@ -0,0 +1,78 @@
+package commands
+
+import (
+	"bytes"
+	"github.com/jfrog/build-info-go/utils"
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/commands/transferfiles/state"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/tests"
+	"github.com/jfrog/jfrog-client-go/pipelines/services"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"github.com/stretchr/testify/assert"
+	"testing"
+	"time"
+)
+
+func initDisplaySyncStatusTest(t *testing.T) (*bytes.Buffer, func()) {
+	cleanUpJfrogHome, err := tests.SetJfrogHome()
+	assert.NoError(t, err)
+
+	// Create transfer directory
+	transferDir, err := coreutils.GetJfrogTransferDir()
+	assert.NoError(t, err)
+	err = utils.CreateDirIfNotExist(transferDir)
+	assert.NoError(t, err)
```

Comment:
The above lines seem redundant and unrelated to this test. Notice they have been copird to other tests as well as part of this PR. Let's remove all of them.